docs/content/faq/how-can-i-deploy-a-docker-image.md

```diff
@@ -0,0 +1,43 @@
+# How can I deploy a Docker image to Flynn?
+
+Flynn supports running Docker containers from Docker registries. Note that currently the volumes of Docker apps are ephemeral and will be wiped when the instance reboots or is scaled down.
```

Comment:
I tend to call things in registries "images", rather than "containers", so I would probably say:

```
Flynn supports running Docker images from Docker registries
```

I also find "Docker apps" confusing (I feel it indicates the images will actually run using Docker), so I would consider rewording to something like:

```
Note that currently any volumes declared in a Docker image are ephemeral...
```